migtests/tests/oracle/bi-db/env.sh

```diff
@@ -1,3 +1,7 @@
 export SOURCE_DB_TYPE="oracle"
 export SOURCE_DB_SCHEMA=${SOURCE_DB_SCHEMA:-"BI"}
 export TARGET_DB_NAME=${TARGET_DB_NAME:-"bi_db_test"}
+export SOURCE_DB_USER_SCHEMA_OWNER=${SOURCE_DB_USER_SCHEMA_OWNER:-"TEST_SCHEMA"}
+export SOURCE_DB_USER_SCHEMA_OWNER_PASSWORD=${SOURCE_DB_USER_SCHEMA_OWNER_PASSWORD:-"password"}
+export SOURCE_DB_USER_SYS=${SOURCE_DB_USER_SYS:-"SYS"}
```

Comment:
Add these to the main `oracle/env.sh` and remove them from test-specific `env.sh`

Reply:
Done